snapcraft/internal/sources.py

```diff
@@ -389,7 +389,13 @@ def get(sourcedir, builddir, options):
     handler_class = _get_source_handler(source_type, options.source)
     handler = handler_class(options.source, sourcedir, source_tag,
                             source_branch)
-    handler.pull()
+    try:
+        handler.pull()
+    except FileNotFoundError:
```

Comment:
yes, this doesn't sound too good. To better identify it we would have to raise an exception of our own on the pull. Like which bzr || raise exception.
So at least make an XXX comment here with a link to the bug. Even better to fix it at once :)